ut/client_ut.cpp

```diff
@@ -1097,6 +1098,52 @@ TEST_P(ClientCase, QuerySettings) {
     EXPECT_THROW(client_->Execute(query), ServerException);
 }
 
+TEST_P(ClientCase, TracingContext) {
+    Block block;
+    createTableWithOneColumn<ColumnString>(block);
+
+    Query query("INSERT INTO " + table_name + " (*) VALUES (\'Foo\'), (\'Bar\')" );
+    open_telemetry::TracingContext tracing_context;
+    std::srand(std::time(0));
+    tracing_context.trace_id.first  = std::rand();
+    tracing_context.trace_id.second = std::rand();
+    query.SetTracingContext(tracing_context);
+    client_->Execute(query);
+
+    try {
```

Comment:
Sine this is not the first place were "SYSTEM FLUSH LOGS" is executed, it makes sense to factor out this code into a function/method, so there are no code duplicates.

Something like:
```
// client_ut.cpp
void FlushLogs(Client& client);

// later in the test:
FLusgLogs(*client_);
```